README.md

```diff
@@ -1,222 +1,74 @@
-
-    ```
-    /tests/visual/components/component-name
-    -- index.html
-    -- component-name.css
-    -- component-name.css.map
-    -- component-name.scss
-    ```
-
-7. Files descriptions:
-
-    Write your SCSS styles in the **/dist/components/_component-name.scss**
-
-    Create and test your markup in the **/tests/visual/index.html**
-
-    When markup is done create its dust template in the **/dist/components/component-name.dust**
-
-    Fill json file with dummy data for dust template in **/dist/components/component-name.json**
+## Previewing Utilities
 
-    Write instucrions how to use this component in **/dist/components/readme.md**
+1. If you have Sass < 3.4, run `gem update sass`. If you don't have Bower installed, run `npm install -g bower`.
 
-## Updating `gh-pages` Online Docs
+2. In your utilities directory, run `npm install && bower install`.
 
-We use the `gh-pages` branch to host Stencil's visual test and make them accessible via the interwebs. In order to maintain these docs, we use [git-subtree](https://github.com/git/git/tree/master/contrib/subtree) to keep the `gh-pages` tests in sync with `master`.
+3. Run `grunt serve` to run testing environment locally. It will automatically recompile your stylesheets when you change them.
 
-Before you update the docs, make sure that `master` is fully up to date with all the latest tests and documentation.
+4. Navigate to localhost:3000.
 
-The process is as follows:
+5. Utilities are found in tests/visual/ directory.
 
-1. Checkout the `master` branch
-2. Run `git subtree split --branch gh-pages --prefix tests/`
-3. Run `git push origin gh-pages`
-3. Done!
+Learn about setting up and running Stencil tests in our [dev centre documentation](/).
 
-What did that do? Well a subtree split effectively takes a sub-directory (in our case `tests/`) and creates a new branch (`gh-pages`) for just that directory. Of course, instead of creating a new branch, we simply pulled over only the commits from master that apply to that directory and added it to the sub-tree branch. That's why when you check out the `gh-pages` branch, the root directory consists of all the files found in the `tests/` directory of `master`.
 
-**WARNING** do NOT directly commit to the `gh-pages` branch! This will break our ability to use the git-subtree tool. Only apply changes to `gh-pages` using the subtree tool.
+## Learn More
 
-In case if you committed directly to GH-Pages you will see errors running `git push origin gh-pages` command. To fix it repeat these steps:
+To learn more about Utilities, Stencil, and Stencil Components, visit our dev centre documentation:
 
-1. Revert your commit and push it (so you have 2 commits in total - one with changes and one that cancels these changes)
-2. Delete the `gh-pages` branch from the Github repo (remote origin)
-2. Run `git push origin gh-pages`
+[Use a Component](/)
+[Tips and Recommendations](/)
+[Uninstall a Component](/)
```

Comment:
If there is no links yet can we add in the brackets something like [Use a Component (link is missing)](/)? It can be forgotten cause you don't see the actual links